src/lib/components/NavConversationItem.svelte

```diff
@@ -7,15 +7,18 @@
 	import CarbonTrashCan from "~icons/carbon/trash-can";
 	import CarbonClose from "~icons/carbon/close";
 	import CarbonEdit from "~icons/carbon/edit";
+	import { useSettingsStore } from "$lib/stores/settings";
 
-	export let conv: { id: string; title: string };
+	export let conv: { id: string; title: string; assistantId?: string; avatarHash?: string };
 
```

Comment:
can we use the type from https://github.com/huggingface/chat-ui/blob/feature/assistants/src/lib/components/NavMenu.svelte#L11 ?

Reply:
good catch I made the type reusable